scripts/file_utils.py

```diff
@@ -13,11 +13,7 @@
 import os
 # CCPP framework imports
 from parse_tools import CCPPError, ParseInternalError
-#XXgoldyXX: v Crap required to support python 2
 import sys
```

Comment:
I believe this `import sys` can be removed as well, as it doesn't look like that module is used for anything else in this file.

Reply:
Yup, VSCode called it out instantly as unused, removed!